### ISSUE TITLE
Wrong version parsing with number in name

### ISSUE THREAD

**Issue by danyeaw** [dvershinin/lastversion#102]
If libraries have a number in their name, they get parsed as part of the version.

```last version -v libssh2/libssh2
lastversion.lastversion - INFO - Verbose 1 level output.
lastversion.ProjectHolder - INFO - Using cache directory: C:\Users\dan\AppData\Local\lastversion\lastversion\Cache.
lastversion.ProjectHolder - INFO - Created instance of GitHubRepoSession
lastversion.GitHubRepoSession - INFO - Using API token.
lastversion.GitHubRepoSession - INFO - Got HTTP status code 200 from https://github.com/libssh2/libssh2/releases.atom
lastversion.GitHubRepoSession - INFO - Checking tag libssh2-1.10.0
lastversion.ProjectHolder - INFO - Sanitizing string libssh2-1.10.0 as a satisfying version.
lastversion.ProjectHolder - INFO - Parsed as Version OK. String representation: 2.1.10.0.
```

I was expecting that the representation should be 1.10.0 instead of 2.1.10.0.

**Comment by dvershinin** [comment]
Was fixed in v3.0.0

**Comment by danyeaw** [comment]
Thanks @dvershinin!!
